runtime/doc/syntax.txt

```diff
@@ -5054,6 +5054,8 @@ QuickFixLine	Current |quickfix| item in the quickfix window. Combined with
 							*hl-Search*
 Search		Last search pattern highlighting (see 'hlsearch').
 		Also used for similar items that need to stand out.
+							*hl-SnippetTabstop*
+SnippetTabstop	Tabstops in snippets.
```

Comment:
```suggestion
SnippetTabstop	Tabstops in snippets. |vim.snippet|
```

Reply:
Maybe link directly to `vim.snippet.expand()` instead?

Reply:
I thought about that, but thought the main module would be simpler here. But either would work.